information-schema.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Information Schema
+summary: CockroachDB represents your cluster's schema in a database called information_schema, which contains read-only views that you can use for introspection into your datbase's tables, columns, indexes, and views.
+toc: false
+---
+
+CockroachDB represents your cluster's schema in a database called `information_schema`, which contains read-only [views](views.html) that you can use for introspection into your database's tables, columns, indexes, and views.
+
+This notion of an information schema is part of the SQL standard, which means it is portable and will remain stable. This contrasts with other objects like `pg_catalog`, which is not part of the SQL standard and handles handles specific implemenation issues.
+
+{{site.data.alerts.callout_info}}The <code>information_schema</code> views typically represent objects that the current user has privilege to access. To ensure you can view your cluster's entire schema, access it as the <code>root</code> user.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Objects
+
+To perform introspection on objects, you can either read from the related `information_schema` view or use one of CockroachDB's `SHOW` statements.
+
+Object | Information Schema View| SHOW 
+-------|--------------|--------
+Columns | [columns](#columns)| [`SHOW COLUMNS`](show-columns.html) 
+Constraints | [key_column_usage](#keycolumnusage), [table_constraints](#tableconstraints)| [`SHOW CONSTRAINTS`](show-constraints.html) 
+Databases | [schemata](#schemata)| [`SHOW DATABASE`](show-database.html) 
+Indexes | [statistics](#statistics)| [`SHOW INDEX`](show-index.html) 
+Privileges | [schema_privileges](#schemaprivileges), [table_privileges](#tableprivileges)| [`SHOW GRANTS`](show-grants.html) 
+Tables | [tables](#tables)| [`SHOW TABLES`](show-tables.html) 
+Views | [tables](#tables), [views](#views)| N/A 
+
+## Views
+
+The `information_schema` database is comprised of many views representing your cluster's schema, each of which is detailed below.
+
+### columns
+
+The `columns` view contains information about the columns in each table.
+
+Column | Description
+-------|-----------
+`TABLE_CATALOG` | Always equal to `def` (CockroachDB does not support the notion of catalogs)
```

Comment:
Add period at end of this and all other descriptions, or (and this may be easier for you) remove periods at end of descriptions.